helix-term/src/ui/prompt.rs

```diff
@@ -34,6 +39,17 @@ pub enum CompletionDirection {
     Backward,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum Movement {
+    BackwardChar(usize),
+    BackwardWord(usize),
+    ForwardChar(usize),
+    ForwardWord(usize),
```

Comment:
I don't think we support repetition for prompt since we only need one. I think we might be able to remove the usize here.

Reply:
Good point, I'll remove it

Reply:
Yeah wezterm also always uses 1, I guess it was added since you could do vi-mode for readline in the future with it

Reply:
Yeah, but I don't think we will be doing that any time soon right? At least I think vi-mode won't be useful in parts like prompt where a modal editor is not useful.